HLTrigger/Timer/plugins/FastTimerService.cc

```diff
@@ -731,7 +731,7 @@ void FastTimerService::PlotsPerJob::book(dqm::reco::DQMStore::IBooker& booker,
     if (bymodule) {
       booker.setCurrentFolder(basedir + "/process " + process.name_ + " modules");
       for (unsigned int id : process.modules_) {
-        auto const& module_name = job.module(id).moduleLabel();
+        auto const& module_name = fix_for_dqm(job.module(id).moduleLabel());
```

Comment:
@fwyzard The change is here.

Reply:
thanks, I guess I missed it !

Reply:
I force pushed the formatting update. Sometimes github does not register force pushed changes.